db/migrations/01_makers_bnb.sql

```diff
@@ -1 +1,2 @@
-CREATE TABLE spaces(id SERIAL PRIMARY KEY, name VARCHAR(60)) ;
+CREATE TABLE spaces(id SERIAL PRIMARY KEY, name VARCHAR(60), description VARCHAR(60), price_per_night DOUBLE PRECISION) ;
```

Comment:
Do we want the description to be limited to the same number of characters as the name?